_posts/2019-07-24-lts-plan.md

```diff
@@ -8,7 +8,7 @@ Today we're formally announcing our [Long Term Support plan](https://github.com/
 
 Starting today, Bootstrap 3 will move to end of life, and will no longer receive critical security updates.
 
-Bootstrap 4 moved to Long Term Support release on February 14, 2019 and will no longer receive new features. It will continue to receive bug fixes, security updates, and documentation updates.
```

Comment:
This might require further changes in the release repo.

Reply:
I'll handle it as necessary, but when are we looking to release 4.4?